.github/workflows/ci.yml

```diff
@@ -0,0 +1,78 @@
+name: PyPi release
+run-name: CI pipeline
+on:
+  push:
+    branches:
+      - '*'
+
+jobs:
+  style:
+    name: Check style
+    runs-on: ubuntu-latest
+    env:
+      TOXENV: flake8,black
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.7"
+
+      - name: deps
+        run: python -m pip install -U tox
+
+      - name: Check style
+        run: tox -vv
+
+  build:
+    name: Check build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.7"
+
+      - name: deps
+        run: python -m pip install -U wheel setuptools build
+
+      - name: Check build
+        run: python setup.py sdist bdist_wheel
+
+  test:
+    name: Check test
+    runs-on: ubuntu-latest
+    env:
+      TOXENV: py37
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.7"
+
+      - name: deps
+        run: python -m pip install -U tox
+
+      - name: Check test
+        run: tox -vv
+
+  documentation:
+    name: Generate documentation
```

Comment:
Probably ok to leave in place, since it is in the current gitlab repo. Even though Jim confirmed that it is not currently being used.

Reply:
Thought it would be useful as it at least checks if the documentation can be generated. If any new documentation it will confirm there are no generation errors.